src/gateway/kong-enterprise/fips-support.md

```diff
@@ -0,0 +1,40 @@
+---
+title: FIPS 140-2
+badge: enterprise
+content_type: reference
+---
+
+{{site.base_gateway}} provides support for the Federal Information Processing Standard (FIPS 140-2). Compliance with this standard is typically required for working with U.S. federal government agencies and their contractors.
+
+{{site.base_gateway}} offers a FIPS compliant package. The package meets FIPS compliancy because the primary library in {{site.base_gateway}}, OpenSSL, is replaced with the [BoringSSL](https://boringssl.googlesource.com/boringssl/), which at its core uses the FIPS 140-2 compliant BoringCrypto for cryptographic operations.
+
```

Comment:
```suggestion

{{site.base_gateway}} uses FIPS compliant algorithms in all core components when configured. Not all plugins are currently not FIPS compliant.
```

Reply:
@fffonion, @mheap - this wasn't included in the Legal review so I've modified to just refer to the use of BoringSSL rather than statements on FIPS compliance.